lib/query.js

```diff
@@ -320,42 +289,28 @@ Query.prototype.select = function (bins) {
  * })
  *
  * @see {@link module:aerospike/filter} to create secondary index filters.
- * @see {@link module:aerospike/predexp} to create predicate expression filters.
  *
  * @example <caption>Applying a predicate expression in addition to a secondary
  * index filter to find only record where the 'tags' list bin contains the
  * value 'blue' but also contains at least one other value that is not
  * 'blue':</caption>
```

Comment:
Example maybe can be removed, since there already is a separate example showing how to apply secondary index filters.